text/0148-sampling-probability.md

```diff
@@ -0,0 +1,744 @@
+Sampling algorithms that do this are known as _weighted sampling
+algorithms_.
+
+#### Merging samples
+
+To merge samples means to combine two or more frames of sample data
+into a single frame of sample data that reflects the combined
+populations of data in an unbiased way.  Two weighted sampling
+algorithms are listed below in [Recommended Reading](#recommended-reading).
+
+In a broader context, weighted sampling algorithms support estimating
+population weights from a sample of unequal-weight items.  In a
+telemetry context, item weights are generally event counts, and
+weighted sampling algorithms support estimating total population
+counts from a sample with unequal-count items.
+
+The output of weighted sampling, in a telemetry context, are samples
+containing events with new, unequal adjusted counts that maintain
+their power to estimate counts in the combined population.
+
+#### Maintaining "Probability proportional to size"
+
+The statistical property being maintained in the definition for
+weighted sampling used above is known as _probability propertional to
+size_.  The "size" of an item, in this context, refers to the
+magnitude of each item's contribution to the total that is being
+estimated. To avoid bias, larger-magnitude items should have a
+proportionally greater probability of being selected in a sample,
+compared with items of smaller magnitide.
+
+The interpretation of "size", therefore, depends on what is being
+estimated.  When sampling events with a natural size, such as for
+Metric Sum data points, the absolute value of the point should be
+multiplied with the adjusted count to form an effective input weight
+(i.e., its "size" or contribution to the population total).  The
+output adjusted count in this case is the output from weighted
+sampling divided by the (absolute) point value.
+
+#### Zero adjusted count
+
+An adjusted count with zero value carries meaningful information,
+specifically that the item participated in a probabilistic sampling
+scheme and was not selected.  A zero value can be be useful to record
+events when they provide useful information despite their effective
+count; we can use this to combine multiple sampling schemes in a
+single stream.
+
+For example, consider collecting two samples from a stream of spans,
+the first sample from those spans with attribute `A=1` and the second
+sample from those spans with attribute `B=2`.  Any span that has both
+of these properties is eligible to be selected for both samples, in
+which case one event could have two non-zero adjusted counts (e.g.,
+`sampling.by_a.adjusted_count` and `sampling.by_b.adjusted_count`).
+
+## Proposed Tracing specification
+
+For the standard OpenTelemetry Span Sampler implementations to support
+a range of probability sampling schemes, this document recommends the
+use of a Span attribute named `sampling.sampler_name.adjusted_count`
+to encode an unbiased adjusted count computed by a Sampler.
```

Comment:
What about situations where sampling is either biased or is subject to after-sampling rate limiting?